libs/ml/include/ml/distributed_learning/distributed_learning_client.hpp

```diff
@@ -46,22 +46,43 @@ struct ClientParams
 {
   using SizeType = fetch::math::SizeType;
 
-  SizeType batch_size;
+  SizeType batch_size{};
   DataType learning_rate;
 
   std::vector<std::string> inputs_names = {"Input"};
   std::string              label_name   = "Label";
   std::string              error_name   = "Error";
 };
 
+template <typename TensorType>
+struct GradientUpdate
```

Comment:
as per prior conversation, this Update struct should be a generic object containing:
- data (where you store your gadients in this case)
- data type flag
- timestamp
- client id
- hash

therefore can you rename it to "Update"
and rename grads to data
and add a comment explaining above explaining what the update struct is for.